kf_lib_data_ingest/app/settings/base.py

```diff
@@ -7,15 +7,24 @@
                                  'target_apis',
                                  'kids_first.py')
 
+
+class ENV_KEYS:
+    KF_STUDY_CREATOR_API_TOKEN = 'KF_STUDY_CREATOR_API_TOKEN'
+    KF_AUTH0_DOMAIN = 'KF_AUTH0_DOMAIN'
```

Comment:
Can you please remove the domain and auth0_aud keys from here? They're not secrets and I'd like to see those in the logs. Let's keep this to sensitive values only.

Reply:
how about now